ml-conversational-analytic-tool/run.py

```diff
@@ -46,6 +66,7 @@ def run(annotated_filename, dataset_filename, outcome, encoding_type, model_type
     parser.add_argument('dataset_filename', help='File location of extracted dataset')
     parser.add_argument('model', help='Model type to use for training, supported CNN and LSTM')
     parser.add_argument('outcome', help='Inclusive, Constructive, or Both')
+    parser.add_argument('-save', help='Save the trained model')
```

Comment:
can you clarify in the help msg what is the type of the argument `-save` expects. For me, at first time it was unclear if `-save` is kind of a boolean and does not expect an additional argument if it is a path or a file name ...